Update to grunt 1.0

This is causing builds to fail since `grunt-wiredep` is using `~0.4.0` whereas most other grunt plugins are using `>=0.4.0`
```
npm WARN peerDependencies The peer dependency grunt@~0.4.0 included from grunt-wiredep will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.

npm ERR! peerinvalid The package grunt@1.0.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer grunt-wiredep@3.0.0 wants grunt@~0.4.0
```

--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
     "npm": ">=2.3.0"
   },
   "peerDependencies": {
-    "grunt": "~0.4.0"
+    "grunt": ">=0.4.0"
   },
   "dependencies": {
     "wiredep": "^4.0.0"

